build.gradle

```diff
@@ -1,19 +1,19 @@
 import com.linkedin.gradle.DistributeTask
 
 buildscript {
-    ext.kotlinVersion = '1.1.2'
+    ext.kotlinVersion = '1.3.61'
 
     repositories {
         jcenter()
+        google()
     }
     dependencies {
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
-        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.4'
-        classpath 'com.github.dcendents:android-maven-gradle-plugin:1.4'
-        classpath 'org.jfrog.buildinfo:build-info-extractor-gradle:4.4.0'
-        classpath 'com.github.jengelman.gradle.plugins:shadow:1.2.4'
-        classpath 'org.jetbrains.dokka:dokka-gradle-plugin:0.9.9'
-        classpath 'com.android.tools.build:gradle:2.3.0'
+        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.4'
+        classpath 'com.vanniktech:gradle-maven-publish-plugin:0.8.0'
+        classpath 'org.jfrog.buildinfo:build-info-extractor-gradle:4.11.0'
+        classpath 'com.github.jengelman.gradle.plugins:shadow:5.1.0'
+        classpath 'com.android.tools.build:gradle:3.5.3'
```

Comment:
Unless these are needed to get the publication,  i would recommend removing them, just because we don't know all the details of what has changed, and these plugins are very hard to test without actually publishing artifacts.

Reply:
we need to update these dependencies to use them with the new Android Gradle Plugin (old versions of them are not compatible with the new AGP)

Reply:
do you want me to revert updating AGP?

Reply:
Hi, apologies again for the delay here. I think we should probably drop this part - I tried testing out publishing flow locally with these changes, and hit some dependency issuese resulting in failures (`Caused by: java.lang.ClassNotFoundException: org.apache.commons.io.IOUtils` from within the artifactory client). I know the publishing flow is hard to test, so I'm thinking its probably easier for this change to just focus on the functional change. Are you ok with that?

Reply:
Thanks!